ballerina/xmldata.bal

```diff
@@ -66,6 +67,9 @@ public isolated function fromJson(json jsonValue, JsonOptions options = {}) retu
             }
         }
     }
+    if !(jsonValue is null) {
```

Comment:
```suggestion
    if jsonValue not is null {
```

Reply:
Are we already handling null values?

Reply:
Currently, an error (`"failed to parse xml"`) is being returned for `null` values. I think this should be changed to return an empty sequence. WDYT.

Reply:
Fixed in 1da75b6